tests/testthat/test-write_camtrapdp.R

```diff
@@ -0,0 +1,93 @@
+test_that("write_camtrapdp() writes datapackage.json and CSV files to a
```

Comment:
I've deviated from the frictionless::write_package() test and aligned it more with write_dwc()